libs/blocks/brand-concierge/brand-concierge.css

```diff
@@ -209,10 +209,18 @@
   line-height: 1.5;
   outline: none;
   width: 100%;
+  resize: none;
+  overflow-wrap: break-word;
+  word-wrap: break-word;
+  white-space: pre-wrap;
```

Comment:
Is the `white-space` declaration coming from a requirement? I'd assume it'd be cleaner without it in certain scenarios. Trying a Cmd + Return already submits the result, so I'm guessing this might be here for when someone pastes something in?